packages/expandable-search-bar/src/expandable-search-bar.ts

```diff
@@ -80,6 +80,19 @@ export default class ExpandableSearchBar extends LitElement {
     `;
   }
 
+  /**
+   * Workaround for lit-element 2.2.1 not reflecting this.searchTerm 
+   * in the HTML element. updated is called when this.searchTerm's value
+   * is changed.
+   *
+   * @memberof ExpandableSearchBar
+   */
+  updated() {
+      if (this.searchInput) {
+        this.searchInput.value = this.searchTerm;
+      }
```

Comment:
The `updated()` method gets called when any of the properties get updated so we'd want to limit this to just when `searchTerm` is updated. You can add the [`(props: PropertyValues)`](https://lit.dev/docs/components/lifecycle/#updated) argument to the method which contains all of the properties that were updated, ie `if (props.has('searchTerm')) { ... }`.

Also, could you reindent to 2 spaces?